pkg/utils/connections/connection.go

```diff
@@ -239,12 +239,22 @@ func saveConnectionsConfigFile(ConnectionConfig *ConnectionConfig) *ConError {
 
 // getConnectionConfigDir : get directory path to the connections file
 func getConnectionConfigDir() string {
-	const GOOS string = runtime.GOOS
+	val, ok := os.LookupEnv("CHE_API_EXTERNAL")
```

Comment:
what is `ok`? If this is a bool, can this var name be conveyed in a better way?

Reply:
ok is a bool for whether the env variable is set or not. val is then the value of the env variable.

Changing to isSet for better clarity.